### PR TITLE
Add orphan volume endpoints

### PR DESCRIPTION
**Pull request by gohilankit**
This MR adds following:
1. Orphan volume endpoints to Swagger yaml. 
2. Required changes in basicauth/oauth2 deployment manifests
3. Increased timeout for /waitforjob API in nginx conf
4. client-sdk for orphan volume APIs as well as missing models in client-sdk
5. Orphan volume API client examples in `basicauth_client.go`.

### CODE REVIEW

File: deploy/swagger-template.yaml [hunk 1]
```diff
@@ -406,6 +408,131 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /orphanvolumes:
```
Comment (kolluria) [inline deploy/swagger-template.yaml:411]: Add the latest modifications that introduce `successfulOrphanDeletions` and `failedOrphanDeletions`, and generate the client-sdk again.

File: client-sdk/go/client/model_orphan_volume_delete_result.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package swagger
+
+type OrphanVolumeDeleteResult struct {
```
Comment (kolluria) [inline client-sdk/go/client/model_orphan_volume_delete_result.go:19]: This is not consistent with the spec. Can you generate it again?

File: client-sdk/go/examples/basicauth_client.go [hunk 3]
```diff
@@ -108,4 +109,34 @@ func main() {
 	// logger.Info("Result", "result", res) //This gives the result in json format
 	// logger.Info("HTTP status", "status", resp.Status)
 	//=======================================================
+
+	//======List orphan volumes======
```
Comment (kolluria) [inline client-sdk/go/examples/basicauth_client.go:113]: I see that some of the examples are commented out while some are not. Can you describe the intent of this?

**Review by kolluria — CHANGES_REQUESTED** [review]
Submitted my review with a few minor comments.

File: client-sdk/go/client/model_orphan_volume_delete_result.go (hunk 2 above)
Comment (gohilankit) [reply client-sdk/go/client/model_orphan_volume_delete_result.go:20]: Regenerated the client-sdk from latest spec.

File: client-sdk/go/examples/basicauth_client.go (hunk 3 above)
Comment (gohilankit) [reply client-sdk/go/examples/basicauth_client.go:113]: Say if you just want to test invocation of a single API, it's easier to uncomment just that API example and execute the code. Do you want to structure it in a different way?

Comment (gohilankit) [reply client-sdk/go/examples/basicauth_client.go:113]: Here's an example of invoking a single endpoint after adjusting the basicauth credentials and server url.

```
agohilZLVDQ:examples agohil$ go run basicauth_client.go 
2023-05-11T16:24:37.109-0700    INFO    Main    examples/basicauth_client.go:123        Result  {"result": {"totalOrphans":0,"orphanVolumes":[],"retryAfterMinutes":8}}
2023-05-11T16:24:37.109-0700    INFO    Main    examples/basicauth_client.go:124        HTTP status    {"status": "200 OK"}
```

The users can copy whatever example they want to incorporate in their code.

Comment (kolluria) [reply client-sdk/go/examples/basicauth_client.go:113]: I see. Makes sense.

File: deploy/oauth2/deploy-template.yaml [hunk 4]
```diff
@@ -415,7 +416,7 @@ spec:
             - name: swagger-api
               mountPath: /api
         - name: cns-manager
-          image: projects.registry.vmware.com/cns_manager/cns-manager:r0.1.1-rc.1
+          image: projects.registry.vmware.com/cns_manager/cns-manager:r0.2.0-rc.6
```
Comment (kolluria) [inline deploy/oauth2/deploy-template.yaml:419]: What's the plan to change the value to the official release image?

**Review by kolluria — APPROVED** [review]
LGTM

File: deploy/oauth2/deploy-template.yaml (hunk 4 above)
Comment (gohilankit) [reply deploy/oauth2/deploy-template.yaml:419]: We'll change both in `master` and a new `release-0.2` branch when the official image is available, before cutting a new release.

**Pull request merged**